mcpgrafana.go

```diff
@@ -111,11 +115,16 @@ var ExtractGrafanaClientFromEnv server.StdioContextFunc = func(ctx context.Conte
 		if url.Scheme == "http" {
 			cfg.Schemes = []string{"http"}
 		}
+	} else {
+		grafanaURL = defaultGrafanaURL
 	}
-	if apiKey := os.Getenv(grafanaAPIEnvVar); apiKey != "" {
+
+	apiKey := os.Getenv(grafanaAPIEnvVar)
+	if apiKey != "" {
 		cfg.APIKey = apiKey
 	}
 
+	slog.Debug("Creating Grafana client", "url", grafanaURL, "api_key_set", apiKey != "")
```

Comment:
We already do the parsing... should probably just save that variable and then use `Redacted()`.